_config.yml

```diff
@@ -11,8 +11,8 @@ discord_link:       https://discord.gg/2tWhanq
 github_link:        https://github.com/UWB-ACM
 linkedin_link:      https://www.linkedin.com/in/uwbacm/
 
-# Add your google-analytics ID here to activate google analytics
-google_analytics:   UA-XXXXXXXXX-X # out your google-analytics code
+# Google-analytics ID
+google_analytics:   UA-170203794-1 
```

Comment:
who owns this GA?

Reply:
good catch! this is my GA id for testing. I removed it in update #2. will replace with ACM's GA id if needed.